include/GLFW/glfw3native.h

```diff
@@ -564,6 +564,11 @@ GLFWAPI int glfwGetOSMesaDepthBuffer(GLFWwindow* window, int* width, int* height
 GLFWAPI OSMesaContext glfwGetOSMesaContext(GLFWwindow* window);
 #endif
 
+#if defined(GLFW_EXPOSE_NATIVE_ANDROID)
+#include <android_native_app_glue.h>
+GLFWAPI struct android_app * glfwGetAndroidApp(GLFWwindow* window);
```

Comment:
There doesn't seem to be a need for a window parameter here as the app struct is available in a global.